src/pages/portfolio/ProjectCard.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Card, CardContent, CardMedia, Typography, Chip, Box, Link, CardActions, Button } from '@mui/material';
+import { Github, Globe } from 'lucide-react';
+import { Project } from '../../types';
+
+interface ProjectCardProps {
+  project: Project;
+}
+
+const ProjectCard: React.FC<ProjectCardProps> = ({ project }) => {
+  return (
+    <Card sx={{ height: '100%', display: 'flex', flexDirection: 'column' }}>
+      <CardMedia
+        component="img"
+        height="200"
+        image={project.imageUrl}
+        alt={project.title}
+      />
+      <CardContent sx={{ flexGrow: 1 }}>
+        <Typography variant="h5" gutterBottom>
+          {project.title}
+        </Typography>
+        <Typography variant="body2" color="text.secondary" paragraph>
+          {project.description}
+        </Typography>
+        <Box sx={{ display: 'flex', gap: 1, flexWrap: 'wrap', mb: 2 }}>
+          {project.technologies.map((tech) => (
+            <Chip key={tech} label={tech} size="small" />
+          ))}
+        </Box>
+      </CardContent>
+      <CardActions>
+        <Button 
+          startIcon={<Github size={18} />}
+          href={project.githubUrl}
+          target="_blank"
+          size="small"
+        >
+          Code
+        </Button>
+        {project.liveUrl && (
```

Comment:
project.liveUrl.length !== 0 && (...)

do this for every one if these ternaries you have


unless its a boolean, do NOT do variable && ... 
or variable ? ... : ...

and if a var can be null or undefined

do this var == null or var != null
instead of var === null or var === undefined or just `var` in if checks, ternaries and whatever. be explicit bout that

for booleans you don't need var === true it can be just var and !var